JavaAssignment/src/avadhoot/Converter.java

```diff
@@ -0,0 +1,22 @@
+package avadhoot;
+
+public class Converter 
+{
+	void display(int sec)
+	{
+		int h=0;
```

Comment:
No need to initialize a local variable with 0 value, instead of directly writing int temp = sec/60;